src/main/kotlin/no/nav/familie/klage/distribusjon/Klageresultat.kt

```diff
@@ -0,0 +1,17 @@
+package no.nav.familie.klage.distribusjon
+
+import no.nav.familie.klage.felles.domain.Sporbar
+import org.springframework.data.annotation.Id
+import org.springframework.data.relational.core.mapping.Embedded
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class Klageresultat(
```

Comment:
Er det en slags KlageDistribusjonKvittering

Reply:
KvitteringFraJournalpostBrevOgKabal 😛

Reply:
DistribusjonResultat

Reply:
☝️ 👇 = brainstorming (ikke nødvendigvis gode forslag)
IverksettingKvittering????
Ring hvis du vil brainstorme sammen :)

Reply:
Liker `DistribusjonResultat` - går for den!